prqlc/prql-compiler/tests/integration/snapshots/integration__fmt@group_all.prql.snap

```diff
@@ -1,11 +1,13 @@
 ---
-source: prql-compiler/tests/integration/main.rs
+source: prqlc/prql-compiler/tests/integration/main.rs
 expression: "# mssql:test\nfrom a=albums\ntake 10\njoin tracks (==album_id)\ngroup {a.album_id, a.title} (aggregate price = ((sum tracks.unit_price)))\nsort album_id\n"
-input_file: prql-compiler/tests/integration/queries/group_all.prql
+input_file: prqlc/prql-compiler/tests/integration/queries/group_all.prql
 ---
 from a = albums
 take 10
 join tracks (==album_id)
-group {a.album_id, a.title} (aggregate price = (sum tracks.unit_price))
+group {a.album_id, a.title} (aggregate price = (
+  sum tracks.unit_price
+))
```

Comment:
Great!

Reply:
This isn't ideal - it could be fine-tuned so it breaks before `aggregate` instead of `sum`.